src/Files.App/Views/ShellPanesPage.xaml.cs

```diff
@@ -491,7 +513,7 @@ private void Pane_GotFocus(object sender, RoutedEventArgs e)
 
 		private void Pane_RightTapped(object sender, RoutedEventArgs e)
 		{
-			if (sender != ActivePane && sender is IShellPage shellPage && shellPage.SlimContentPage is not ColumnsLayoutPage)
+			if (sender != ActivePane)
```

Comment:
Why did you delete these conditions? It doesn't appear to be directly related to this PR.

Reply:
Are they needed?

Reply:
I think they are for focusing not a pane but a column when in the column layout.

Reply:
When it's column view, even if you focus background wouldn't changed. You can try in preview build.

Reply:
That is a different issue, and this change causes the focus to flicker when right-clicking a column. I think this change should be reverted.
We have had similar issues before. https://github.com/files-community/Files/issues/12285